ydb/core/scheme/scheme_tablecell.h

```diff
@@ -466,41 +468,16 @@ class TSerializedCellVec {
     }
 
     void Parse(const TString &buf) {
-        Y_VERIFY(TryParse(buf, *this));
+        Y_VERIFY(DoTryParse(buf));
     }
 
     TConstArrayRef<TCell> GetCells() const {
         return Cells;
     }
 
-    static void Serialize(TString& res, const TConstArrayRef<TCell>& cells) {
-        size_t sz = sizeof(ui16);
-        for (auto& c : cells) {
-            sz += sizeof(TValue) + c.Size();
-        }
-
-        if (res.capacity() < sz) {
-            res.reserve(FastClp2(sz + 1) - 1);
-        }
-
-        DoSerialize(res, cells);
-    }
-
-    static TString Serialize(const TConstArrayRef<TCell>& cells) {
-        if (cells.empty())
-            return TString();
-
-        size_t sz = sizeof(ui16);
-        for (auto& c : cells) {
-            sz += sizeof(TValue) + c.Size();
-        }
-
-        TString res;
-        res.reserve(sz);
+    static void Serialize(TString& res, const TConstArrayRef<TCell>& cells);
```

Comment:
`TConstArrayRef` must be passed by value, it's a simple struct with ptr and size and you don't want double pointer indirection.

Reply:
Updated.